### ISSUE TITLE
Factory Manager does not see connected Block Gates.

### ISSUE THREAD

**Issue by Giovanni-NL** [gigabit101/StevesFactoryManager#8]
I'm trying to build this setup https://www.youtube.com/watch?v=XGscDHRm3YM

Problem is that when editing a variable and selecting 'Containers' I can see all connected containers such as chests, quantum storage and drawers, but none of the connected Block Gates. This is not the same behaviour as SFM for minecraft 1.7.10

And yes, I did make sure all blocks are connected using Inventory Cable

**Comment by gigabit101** [comment]
sfm version??

**Comment by Giovanni-NL** [comment]
Doh! forgot to mention the version.  When looking in the MC options it says 1.0.3, however the client folder shows the StevesFactoryManager-1.10.2-1.0.4.jar file.  I'll ask our server op if there might be a version mismatch between our server and our client.

**Comment by gigabit101** [comment]
well there was a bunch of bug fix's in newer version so can you test again after a update and let me no going to close this for now

**Comment by Giovanni-NL** [comment]
will do

**Comment by Giovanni-NL** [comment]
![image](https://cloud.githubusercontent.com/assets/4085882/20273446/a1062802-aa91-11e6-8518-cc7a44928dcf.png)
![image](https://cloud.githubusercontent.com/assets/4085882/20273470/b1339728-aa91-11e6-8ebe-8999ec904f7c.png)
Tested with SFM 1.0.6 this time.  Still the same problem. Block Gate, Inventory Relay and Advanced Inventory Relay are not recognized by the Factory Manager when connected with Inventory Cable. Other types of blocks are ok.  We've tested with only SFM and JEI mods installed, so it's most probably not an issue related to other mods.
